api/src/main/kotlin/edu/wgu/osmt/csv/BatchImportRichSkill.kt

```diff
@@ -175,7 +175,7 @@ class BatchImportRichSkill: CsvImport<RichSkillRow> {
                     keywords = allKeyWords?.let { ListFieldUpdate(add = it) },
                     collections = collections?.let {ListFieldUpdate(add = it)},
                     jobCodes = allJobcodes?.let { ListFieldUpdate(add = it) },
-                    author = NullableFieldUpdate(keywordRepository.getDefaultAuthor())
+                    authors = ListFieldUpdate(add = listOf(keywordRepository.getDefaultAuthor()))
```

Comment:
When I started reviewing, I wondered if this would be updated as well. I appreciate the completeness! :)

Reply:
> @trs-aaron and @ottonomy , this change brings a change to our OSMT API. The OpenAPI spec file should be updated as well.
> 
> We also need to document the impact on existing OSMT instances that deploy this. Is it a new API version? Will this break 3rd party integrations? To clarify, we did agree that WGU would do the house keeping on the older API files.

@JohnKallies I am doing this and setting to new major version as we discussed. I wasn't sure when to commit this since there is the change to `categories` as well. What I planned was to create a new branch with the OpenAPI & CSV specs with both changes. Then you all could also add changes from the filters work, and any other changes planned, and release the specs all at once.

Meant to have that done last night, but I didn't feel well yesterday afternoon. Its what I'm working on this morning.